cli/dyo.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	cli "github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Name:     "dyo",
+		Version:  "0.1.1",
+		Compiled: time.Now(),
```

Comment:
This evaluates at runtime, so Compiled will be now every time the CLI is run :)

Reply:
fixed